invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities-carousel/override.js

```diff
@@ -0,0 +1,7 @@
+// This file is part of CDS RDM
+// Copyright (C) 2022 CERN.
+//
+// CDS RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+export const overriddenComponents = {};
```

Comment:
could you fix the warnings in the file below ?

Reply:
I'm failing to see the warnings.

![Screenshot from 2022-12-07 10-55-37](https://user-images.githubusercontent.com/25476209/206147453-5ff90ea1-a7d4-42ed-86e7-4a4e24a6afed.png)